docs/functions/print_cef.md

```diff
@@ -0,0 +1,113 @@
+---
+title: print_cef
+category: Printing
+example: "{}.print_cef()"
```

Comment:
Can you use a better example here that doesn't just use an empty record?

The same applies to `print_leef`.